notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">I will handle labeled data by encoding with missing data type.
</pre><p>You should mention that the "<em>transformation</em>" process will handle the labeled features.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>